### DISCUSSION TITLE
How can I create a direct reply option for a list that users can click on, similar to a conversation opener?

### DISCUSSION THREAD

**Discussion by CongTang** [langgenius/dify#11172] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Hi All,

I am trying to enable Dify to reply with a list of options, similar to the Conversation Opener. Please see the example below:
![image](https://github.com/user-attachments/assets/73d7d5a5-96f5-4580-9327-83ba77eae168)

I would like users to be able to click on the options to reply directly. Does the Dify "Direct Reply" module have this kind of functionality?

Any help or workaround would be greatly appreciated.

Kind regards,
Cong

**Comment by khoi03** [comment]
I have the same problem. Anyone has a solution?

**Comment by freebeing-Github** [comment]
I have the same requirement here. Can anybody help?

**Comment by vvhr** [comment]
纯dify实现的话使用模板转换节点, 通过jinja语法编写python in html代码
希望更灵活控制组件样式和功能的话, 通过赋值会话变量, 让外部程序通过会话变量的参数来渲染自定义控件和功能
